sklearn/metrics/tests/test_classification.py

```diff
@@ -379,8 +380,7 @@ def test_matthews_corrcoef():
 
     # And also for any other vector with 0 variance
     mcc = assert_warns_message(RuntimeWarning, 'invalid value encountered',
-                               matthews_corrcoef, y_true,
-                               rng.randint(-100, 100) * np.ones(20, dtype=int))
```

Comment:
Similar to the previous change, `y_true` is an array of 'a' and 'b' and we are trying to compare it to an array of ints, so something somewhere should tell us that something is wrong.